tests/test_helpers/test_commands.py

```diff
@@ -312,6 +312,28 @@ def test_localcommands_run(
     assert patched_subprocess.Popen.mock_calls == expected_calls
 
 
+@patch('invenio_cli.helpers.commands.subprocess')
+@patch('invenio_cli.helpers.docker_helper.subprocess')
+def test_localcommands_stop(
```

Comment:
moderate: There is no assertion in this test, in case something does not work at desired what kind of feedback do we get when running tests? How is it notified? stacktrace?

Reply:
The new commit addressed the missing assertion https://github.com/inveniosoftware/invenio-cli/pull/157/commits/63801446b68c62d7a896981a2a7b17eb22d41afd